ddtrace/llmobs/_llmobs.py

```diff
@@ -603,6 +604,11 @@ def enable(
 
             # override the default _instance with a new tracer
             cls._instance = cls(tracer=_tracer, span_processor=span_processor)
+
+            # Add APM trace filter to drop all APM traces when DD_APM_TRACING_ENABLED is falsy
+            apm_filter = APMTracingEnabledFilter()
+            cls._instance.tracer._span_aggregator.user_processors.append(apm_filter)
```

Comment:
I thiiiiiink this looks good, but I remember when we were previously configuring a custom LLMObsTraceProcessor to do a similar thing we had issues on forking due to the trace processor being linked to the span writer and had to do extra logic in the `_child_after_fork()` to configure it correctly to persist after forks. Should we be worried about the same thing here? 

I think this might be a non-issue since the tracer resets the SpanAggregator on fork (which should also keep the existing user processors as is) and we don't rely on `tracer.configure()` here nor is this a TraceFilter (since we're relying on SpanAggregator.user_processors) so I'm less confident that this is a necessary issue but wanted to bring it up just in case.